ITEST/itest.storelogdata/test/org/openl/itest/RunStoreLogDataITest.java

```diff
@@ -749,73 +685,101 @@ public void testKafkaHeaderAnnotations() throws Exception {
+                    assertNotNull(row.getString(CassandraFields.ID));
+                    assertEquals(REQUEST, row.getString(CassandraFields.REQUEST));
+                    assertEquals(RESPONSE, row.getString(CassandraFields.RESPONSE));
+                    assertEquals(HELLO_METHOD_NAME, row.getString(CassandraFields.METHOD_NAME));
+                    assertEquals(SIMPLE4_SERVICE_NAME, row.getString(CassandraFields.SERVICE_NAME));
+                    assertNotNull(row.getTimestamp(CassandraFields.INCOMING_TIME));
+                    assertNotNull(row.getTimestamp(CassandraFields.OUTCOMING_TIME));
+                    assertEquals(PublisherType.KAFKA.toString(), row.getString(CassandraFields.PUBLISHER_TYPE));
+
+                    assertEquals(HELLO_METHOD_NAME, row.getString(CassandraFields.HEADER1));
+                    assertEquals("testHeaderValue", row.getString(CassandraFields.HEADER2));
 
-                return true;
-            }, equalTo(true));
+                    return true;
+                }, equalTo(true));
 
         given().ignoreException(InvalidQueryException.class)
-            .await()
-            .atMost(AWAIT_TIMEOUT, TimeUnit.SECONDS)
-            .until(() -> {
-                final String query = "SELECT * FROM " + h2HelloEntity1TableName;
-                try (Statement stmt = h2Connection.createStatement()) {
-                    java.sql.ResultSet rs = stmt.executeQuery(query);
-                    int count = 0;
-                    while (rs.next()) {
-                        count++;
-                        if (count > 1) {
-                            break;
+                .await()
+                .atMost(AWAIT_TIMEOUT, TimeUnit.SECONDS)
+                .until(() -> {
+                    final String query = "SELECT * FROM " + h2HelloEntity1TableName;
+                    try (Statement stmt = h2Connection.createStatement()) {
+                        java.sql.ResultSet rs = stmt.executeQuery(query);
+                        int count = 0;
+                        while (rs.next()) {
+                            count++;
+                            if (count > 1) {
+                                break;
+                            }
+                            assertEquals(REQUEST, rs.getString(DBFields.REQUEST));
+                            assertEquals(RESPONSE, rs.getString(DBFields.RESPONSE));
+                            assertEquals(HELLO_METHOD_NAME, rs.getString(DBFields.METHOD_NAME));
+                            assertEquals(SIMPLE4_SERVICE_NAME, rs.getString(DBFields.SERVICE_NAME));
+                            assertNotNull(rs.getTimestamp(DBFields.INCOMING_TIME));
+                            assertNotNull(rs.getTimestamp(DBFields.OUTCOMING_TIME));
+                            assertEquals(PublisherType.KAFKA.toString(), rs.getString(DBFields.PUBLISHER_TYPE));
+
+                            assertEquals(HELLO_METHOD_NAME, rs.getString(DBFields.HEADER1));

```

Comment:
!!!! Do formatting of the code in the final commit after approve logical changes. It is difficult to review and merge if conflicts appear. Revert formatting, please in all files.